### ISSUE TITLE
Add luminance threshold value to BloomPass

### ISSUE THREAD

**Issue by fallenartist** [mrdoob/three.js#6138]
I believe there's one important setting missing in BloomPass effect, a luminance threshold, which would apply the effect only above certain relative luminance value. 

This would enable for more realistic effects, where only (very) bright areas of the image are affected, like here: http://en.wikipedia.org/wiki/Bloom_(shader_effect)#mediaviewer/File:Elephants_Dream_-_Emo_and_Proog.jpg

It could be a value between 0 (whole luminance range of image affected, as is now) and 255 (effect applied only to pixels with maximum relative luminance).

edit: I think I meant luminance, not luminosity:
http://en.wikipedia.org/wiki/Relative_luminance
http://stackoverflow.com/questions/596216/formula-to-determine-brightness-of-rgb-color

**Comment by fallenartist** [comment]
Or maybe a better approach would be for blending to happen in Effect Composer, where each effect could be individually controlled, [similarly to how layer blending works in Photoshop](https://helpx.adobe.com/photoshop/using/layer-opacity-blending.html#specify_a_tonal_range_for_blending_layers)?

![ps_layerblendingoptions](https://cloud.githubusercontent.com/assets/661820/6393378/677bf094-bdc1-11e4-9ae2-91ab35c20346.png)

```javascript
effectBloom = new THREE.BloomPass( 1.5, 25, 8, 256 );

composer.addPass({
    effect: effectBloom,
    blending: THREE.AdditiveBlending,
    blendingChannels: "RGBA",
    blendingLumRange: [ 240, 255 ]
});
```

Sorry, I'm not a programmer but you get the idea.

**Comment by bhouston** [comment]
I noticed this issue with the Bloom pass as well.  But what I did was switch my rendering mode to HDR and then bloom worked well with low blur values (like 0.5).  I think the issue you are running into is because you are doing LDR rendering.  I'd suggest just switching to HDR.

**Comment by bhouston** [comment]
I did some research [1] and Bloom is not a spread based on intensity alone as I stated in the previous comment, but rather an artifact of CCD cell overload.  Only when light is outside of the range of a CCD cell, does the bloom occur at all.  Thus it should be implemented based on a luminance threshold and currently it isn't a threshold.

[1] Chapter "10.12 Lense Flare and Bloom" of "Real-Time Rendering 3rd Edition"
